src/globalComponents/Navbar.js

```diff
@@ -70,6 +71,11 @@ class Navbar extends React.Component<Props, State> {
 		window.location = verifyLink('/Login');
 	}
 
+	//Sends the user to the youtube link provided
+	sendUserToYoutubeLink(): void {
+		window.open(this.props.youtubeLink);

```

Comment:
It would be very nice if instead of sending someone to youtube, we embed this video in a popup for example. Embedding a youtube video in a webpage is very easy to do, as well. I'm sure tutorials for it are all over the internet, but here is an example (you can just read the HTML, I didn't even code this in react):
http://wumbogames.github.io/cyberWars/cyberwars.html